canary/apps/react/App.js

```diff
@@ -1,21 +0,0 @@
-import {
```

Comment:
🤦‍♂️

Reply:
For a minute I thought this was in the `react/src` not just in the react subfolder

Reply:
LGTM